src/current/cockroachcloud/resource-usage.md

```diff
@@ -7,11 +7,13 @@ docs_area: deploy
 
 This page describes how to understand your CockroachDB {{ site.data.products.basic }} or {{ site.data.products.standard }} cluster's Request Unit (RU) consumption and tune your workload to balance performance and costs.
 
-- [Understand resource consumption (#understand-resource-consumption) gives an overview of the CockroachDB {{ site.data.products.basic }} and {{ site.data.products.standard }} architecture that affects RU and storage consumption and explains how RUs are calculated.
+- [Understand resource consumption](#understand-resource-consumption) gives an overview of the CockroachDB {{ site.data.products.basic }} and {{ site.data.products.standard }} architecture that affects RU and storage consumption and explains how RUs are calculated.
 
-- [Diagnose and optimize your resource consumption](#diagnose-and-optimize-your-resource-consumption) explains how to find and optimize queries and processes that may be consuming excessive resources and provides general tips for reducing resource usage.
+- [Diagnose and optimize your resource consumption](#diagnose-and-optimize-your-resource-consumption) explains how to find and optimize queries and processes that may be consuming excessive resources
 
-For information on planning your cluster configuration and setting resource limits, refer to [Plan a {{ site.data.products.basic }} Cluster]({% link cockroachcloud/plan-your-cluster-basic.md %}) for a {{ site.data.products.basic }} cluster and [Plan a {{ site.data.products.standard }} Cluster]({% link cockroachcloud/plan-your-cluster.md %}) for a {{ site.data.products.standard }} cluster.
+- [General tips for reducing RU usage](#general-tips-for-reducing-ru-usage) gives recommendations depending on the kind of resource usage.
+
+For information on planning your cluster configuration, refer to [Plan a {{ site.data.products.basic }} Cluster]({% link cockroachcloud/plan-your-cluster-basic.md %}) for a {{ site.data.products.basic }} cluster and [Plan a {{ site.data.products.standard }} Cluster]({% link cockroachcloud/plan-your-cluster.md %}) for a {{ site.data.products.standard }} cluster.
```

Comment:
this sentence could be simplified to 

> refer to 'Plan a Basic Cluster' or 'Plan a Standard Cluster'

since the names are already in the links